contributed_definitions/nyaml/NXcoordinate_system_set.yaml

```diff
@@ -88,138 +88,174 @@ doc: |
-#            has a depends_on field which resolves the possible ambiguities which specific
-#            coordinate systems is referred to. The benefit of this explicit and clear
-#            specifying of the coordinate system used in every case is that especially
-#            in combination with having coordinate systems inside deeper branches
-#            makes up for a very versatile, backwards compatible, but powerful system
-#            to express different types of coordinate systems using NeXus. In the case
-#            of two or more instances of NXcoordinate_system in one :ref:`NXcoordinate_system_set`,
-#            it is also advised to specify the relationship between the two coordinate systems by
-#            using the (NXtransformations) group within NXcoordinate_system.
-#          
-#          In effect, 1 should be the preferred choice. However, if more than one coordinate
-#          system is defined for practical purposes, explicit depends_on fields should
-#          always guide the user for each group and field which of the coordinate system
-#          one refers to.
-#     </doc>
-#     <group type="NXcoordinate_system"/>
-#     <!--How to interpret "depends_on" calls for NXtransformations instances which live
-# within an instance of NXcoordinate_system(_set)?
-# Depends on cardinality of NXcoordinate_system_set, i.e. how many NXcoordinate_system are defined?
-# - 0, the root of an NXtransformation chain i.e. "." means McStas
-# - 1, the root of an NXtransformation chain i.e. "." means "that specific CS" i.e. the one defined by the single instance of NXcoordinate_system in the set
-# - > 1, "." becomes ambiguous. My suggestion, spell out the name of the specific NXcoordinate_system instance to be used, using "."
-# despite all these, a warning should be raised, like a logic warning (and McStas kick in) or an error (i.e. be forbidden) be raised-->
-# </definition>
+  # frequently used coordinate systems with frequently relevant specializations follow
+  # convention 1 of DOI: 10.1088/0965-0393/23/8/083501 is implemented by inheriting type from NXcoordinate_system
+  processing_reference_frame(NXcoordinate_system):
+    doc: |
+      Details about eventually relevant named directions that may give reasons for anisotropies.
+      The classical example is mechanical processing where one has to specify which directions
+      (e.g. rolling, transverse, and normal direction) align how with the direction of the base
+      vectors of the sample_reference_frame.
+      
+      Reference DOI: 10.1016/j.matchar.2016.04.008 suggest to label the
+      base vectors of this coordinate system as Xp, Yp, Zp.
+    origin(NX_CHAR):  # specialized the inherited concept origin from NXcoordinate_system
+      doc: |
+        Location of the origin of the processing_reference_frame.
+        
+        We assume regions-of-interest in this reference frame form a
```

Comment:
I would also drop the "we".

How about "It is assumed ..."

Reply:
Implemented